lib/docker/docker_image.js

```diff
@@ -0,0 +1,191 @@
+import Debug from 'debug';
```

Comment:
Most of this is a combination of lib/docker_image.js and lib/pull_image_stream_to.js